lib/CL/devices/level0/pocl-level0.cc

```diff
@@ -1901,3 +1915,28 @@ cl_int pocl_level0_set_kernel_exec_info_ext(
     return CL_INVALID_VALUE;
   }
 }
+
+cl_int
+pocl_level0_create_finalized_command_buffer(cl_device_id Dev,
+                                            cl_command_buffer_khr CmdBuf) {
+  Level0Device *Device = (Level0Device *)Dev->data;
+  CmdBuf->data[Dev->dev_id] = nullptr;
+  void *CmdBufData = Device->createCmdBuf(CmdBuf);
+  POCL_RETURN_ERROR_ON(
+      (CmdBufData == nullptr), CL_INVALID_VALUE,
```

Comment:
Is this the error code to be returned from `clFinalizeCommandBufferKHR` call? `CL_INVALID_VALUE` is not listed as one of the possible error codes. Perhaps the error code should be `CL_OUT_OF_RESOURCES`?

Reply:
fixed